source/docs/software/wpilib-tools/axon/using-on-a-robot.rst

```diff
@@ -0,0 +1,50 @@
+Using Inference Output on a Robot
+=================================
+
+The provided script for running inference on a coprocessor outputs all inference data over NetworkTables for ease of access.
+
+Output Format
+-------------
+
+Because there can be any number of detected objects in a given image, the inference data is sent in a JSON string to the ``ML/detections`` NetworkTables entry. By compiling all data into a single entry, it is certain that the labels match with the bounding boxes, which may not be true if these items were separate entries.
+The JSON string has the following format:
```

Comment:
```suggestion
Because there can be any number of detected objects in a given image, the inference data is sent in a JSON string to the ``ML/detections`` NetworkTables entry. By compiling all data into a single entry, it is certain that the labels match with the bounding boxes, which may not be true if these items were separate entries.

The JSON string has the following format:
```